data/script/chassiscontrol

```diff
@@ -92,19 +92,19 @@ do_set() {
             pid=`pidof qemu-system-x86_64`
```

Comment:
Would it be randomly return PID of one of qemu processes, considering that multiple of them is running? Then probably one of if statement will fail and cause failure of chassis control failure?